hm_1.py

```diff
@@ -0,0 +1,5 @@
+num = input('Num: ')
```

Comment:
Please add a few examples with different numbers in your code instead of using `input` function.
E.g.:

    first_example = "1239188203011"
    second_example = "123&211*12"

Reply:
Added three variables